Mathlib/GroupTheory/Exponent.lean

```diff
@@ -213,7 +221,7 @@ theorem exponent_dvd_iff_forall_pow_eq_one {n : ℕ} : exponent G ∣ n ↔ ∀
       exact hG
     exact h₂.not_le h₃
 
-@[to_additive (attr := deprecated)]
+@[to_additive]
 theorem exponent_dvd_of_forall_pow_eq_one (n : ℕ) (hG : ∀ g : G, g ^ n = 1) :
     exponent G ∣ n :=
   (exponent_dvd_iff_forall_pow_eq_one G).mpr hG
```

Comment:
Can you make that an `alias` of `exponent_dvd_iff_forall_pow_eq_one` (and make `G` implicit in `exponent_dvd_iff_forall_pow_eq_one`)?